### DISCUSSION TITLE
http://aM.JesusGod-Pope666.Info try out the backgrounds :)

### DISCUSSION THREAD

**Discussion by JesusGod-Pope666-Info** [hvianna/audioMotion.js#41] [Show and tell]
Have taken me a good amount of time, now I know things still can be made a lot better and all - but 20 backgrounds is now in the player at aM.JesusGod-Pope666.Info some music, and Radio channels as well and such.
The Worldly music file that had over 700 radio channels I think is near all sorted for broken and is at around 300 channels now for use. Then I have around 30 proclaimed Christian Channels, hack the planet. But all of them should work - some are censored in the world and other things, so as said, tedious hack the planet as it goes these days.

aM.JesusGod-Pope666.Info/music and aM.JesusGod-Pope666.Info/backgrounds for download of files.

Not sure if you can do a aM.JesusGod-Pope666.Info and get the same kind of filebrowser, any way to bypass the index file to do so? Like a command of some kind? If so it should be possible as well to go into the root of it and have a look from there as that is where the .htaccess file is and starts working from with the command in it: Options +Indexes

Anyway, take a look as long as it is up and use whatever, or get some ideas from it.

-- Darkijah

**Comment by JesusGod-Pope666-Info** [comment]
![Screenshot_2023-10-13_08-36-54](https://github.com/hvianna/audioMotion.js/assets/90806094/f59dd15e-1bb6-4e24-8824-4737e7c69683)
![Screenshot_2023-10-13_08-37-24](https://github.com/hvianna/audioMotion.js/assets/90806094/c5b42b53-0edb-45d8-9b6f-8060e00f3982)
![Screenshot_2023-10-13_08-38-36](https://github.com/hvianna/audioMotion.js/assets/90806094/b2c8d40c-0386-4a61-8b84-bfac8f00cf1a)
![Screenshot_2023-10-13_08-40-19](https://github.com/hvianna/audioMotion.js/assets/90806094/fb0ebad7-b294-45a7-a0c3-e9459610b25a)
![Screenshot_2023-10-13_08-56-27](https://github.com/hvianna/audioMotion.js/assets/90806094/5165e679-69c5-4a5c-b18d-dbdb61c5aa44)

**Comment by hvianna** [comment]
Ooh great collection of background videos!

**Comment by JesusGod-Pope666-Info** [comment]
So I have ingrained the data from the standard media folder with .htaccess - still testing stuff out and things to do.

But overall, anyone can download the data directly here if there is anything they want: http://jesusgod-pope666.info/content/Media/

So the Music folder is now connected to this Media folder, and it works although still testing. MP4 is not shown as it is the old version - so at some point when the new version of audioMotion the MP4 should show up.

Backgrounds have issues loading in httpS mode and don't work. security nonsense as it goes on so many levels making things hard and wasting the time of the user on so many levels. Cors.... httpS and what not.
